modules/nf-neuro/preproc/topup/meta.yml

```
@@ -76,15 +76,18 @@ args:
       type: string
       description: Encoding direction of the forward DWI.
       pattern: "x, y or z"
```

```suggestion
      choices: "x, y or z"
```
I feel choices are more appropriate here than pattern, what do you think?